.ci/cico_do_docker_build_tag_push.sh

```diff
@@ -53,7 +53,12 @@ for distribution in `echo ${ABSOLUTE_PATH}/../${distPath}`; do
   fi
   
   echo "Upstream Eclipse Che version: ${CHE_VERSION}"
-  docker build -t ${DOCKER_IMAGE_URL}:${TAG} --build-arg CHE_DASHBOARD_VERSION=${CHE_VERSION} --build-arg CHE_WORKSPACE_LOADER_VERSION=${CHE_VERSION} -f $DIR/${DOCKERFILE} . | cat
+  if [[ "$CHE_VERSION" == *"SNAPSHOT"  ]]; then
```

Comment:
Should this be "*SNAPSHOT"? or will this find what you want?

Reply:
Che version is something like `7.19.0-SNAPSHOT` so I wanted to create a regex meaning "anything ending SNAPSHOT" - I thought this is the right way to do that.